README.md

```diff
@@ -8,17 +8,24 @@
 This micro service saves sample information from external LIMS into a mongodb instance for easy
 querying.
 
+## Table of contents
+
 <!-- toc -->
 
-- [Requirements](#requirements)
-- [Running](#running)
-- [Updating the MLWH lighthouse_sample table](#updating-the-mlwh-lighthouse_sample-table)
-- [Testing](#testing)
-  - [Testing requirements](#testing-requirements)
-  - [Running tests](#running-tests)
-- [Mypy](#mypy)
-- [Miscellaneous](#miscellaneous)
-  - [Naming conventions](#naming-conventions)
+* [Requirements](#requirements)
+* [Running](#running)
+* [Migrations](#migrations)
+  * [Updating the MLWH lighthouse_sample table](#updating-the-mlwh-lighthouse_sample-table)
+  * [Filtered Positive Rules](#filtered-positive-rules)
+    * [Version 1 `v1` - **Current Version**](#version-1-v1---current-version)
+    * [Propagating Filtered Positive version changes to MongoDB, MLWH and DART](#propagating-filtered-positive-version-changes-to-mongodb-mlwh-and-dart)
```

Comment:
Missing documentation on new migration? Also worth removing the MLWH-only one, if the new migration is going to replace it?

Reply:
I've added the new one and thought of keeping the MLWH one until DART is up and running so that we have a fall-back in case we need it.